resources/lang/en/book.php

```diff
@@ -26,5 +26,7 @@
     'confirm' => [
         'title' => 'Confirm',
         'delete' => 'Do you want to delete this book'
-    ]
+    ],
+    'message' => 'Sorry, Not Be Found.'
```

Comment:
`message` đâu liên quan chi tới `Sorry, Not Be Found.`
đổi tên hoặc chuyển sang file tên `common` đi Luân
'message' => [
'not_found' => 'Sorry, Not Be Found'
]